pom.xml

```diff
@@ -42,7 +42,7 @@
   <licenses>
     <license>
       <name>Apache License, Version 2.0</name>
-      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <url>http://www.apache.org/licenses/LICENSE-${javax.ws.rs-api.version}.txt</url>
```

Comment:
My point is though, maybe we want to be more restrictive with the license. This code is not 'framework' code. I think we don't really want to compete with another vendor who just takes the code and does not want contribute to the project.